src/js/utils/SchemaUtil.js

```diff
@@ -61,7 +61,7 @@ function schemaToFieldDefinition(fieldName, fieldProps, formParent, isRequired,
     definition.checked = value;
   }
 
-  if (fieldProps.fieldType === 'number') {
+  if (fieldProps.type === 'number') {
```

Comment:
Just want to make sure all the schemas under `/service-schemas/**` adhere to this :)